service/storage_service.cc

```diff
@@ -4351,7 +4351,7 @@ future<> storage_service::raft_check_and_repair_cdc_streams() {
 
```

Comment:
Is there a reason why you didn't add a test for `cdc_streams_check_and_repair` and other REST api operations? Is it hard to test or not needed?

Reply:
There are a lot of places where this PR adds timeouts, adding a test for each of them would add too much CI overhead with little value - the changes are trivial.